src/open_clip/factory.py

```diff
@@ -263,18 +285,20 @@ def create_model_from_pretrained(
         force_quick_gelu=force_quick_gelu,
         force_custom_text=force_custom_text,
         cache_dir=cache_dir,
+        text_to_text=text_to_text,
     )
 
     if not return_transform:
         return model
 
-    image_mean = image_mean or getattr(model.visual, 'image_mean', None)
-    image_std = image_std or getattr(model.visual, 'image_std', None)
-    preprocess = image_transform(
-        model.visual.image_size,
-        is_train=False,
-        mean=image_mean,
-        std=image_std
-    )
+    if not text_to_text:
```

Comment:
what about checking if model.visual exists instead ?

Reply:
If we merge `TextTextCLIP` to `CustomCLIP` to get a more general model, then `model.visual` might not exist at all?